db/seeds.rb

```diff
@@ -120,6 +151,10 @@
                   tenant_id: bridge_builders3.id
                   )
 
+debeers_conflict_diamonds_ivorycoast.photos << Photo.create!(
+        image: File.new( "#{Rails.root}/app/assets/images/debeers_diamonds.jpg")
```

Comment:
Indent the first parameter one step more than the previous line.<br>Space inside parentheses detected.